en/docs/reference/config-catalog.md

```diff
@@ -921,6 +921,108 @@ claims_extractor_impl = "org.wso2.carbon.apimgt.impl.token.DefaultClaimsRetrieve
                             </div>
                         </div>
                     </div>
+                    <div class="mb-config">

```

Comment:
Examples are missing as a result all the code elements do not have the "View Sample" option.

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/2051/commits/dc997e2a38619921c44f7ef97eb79bbb3c56bdfe